app/src/main/java/com/example/calculator/MainActivity.kt

```diff
@@ -0,0 +1,101 @@
+package com.example.calculator
+
+import androidx.appcompat.app.AppCompatActivity
+import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import android.widget.Toast
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity() {
+    private var calculate = Calculate()
+    private var inputText = ""
+    private var canAddOperation : Boolean = false
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+    }
+
+    fun numberAction(view: View) {
```

Comment:
Đối với những hàm callback, đặt tên bắt đầu từ on 
```suggestion
    fun onClickNumber(view: View) {
```